### ISSUE TITLE
ffmpeg breaks when playing video with external subs on Roku client

### ISSUE THREAD

**Issue by helpimnotdrowning** [jellyfin/jellyfin#7942]
**Media Info of the file**

Video:
```
General
Complete name                            : W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.mp4
Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom (isom/iso2/avc1/mp41)
File size                                : 1.35 GiB
Duration                                 : 24 min 9 s
Overall bit rate                         : 7 980 kb/s
Writing application                      : Lavf58.78.100

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : High@L4
Format settings                          : CABAC / 4 Ref Frames
Format settings, CABAC                   : Yes
Format settings, Reference frames        : 4 frames
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 24 min 9 s
Bit rate                                 : 7 978 kb/s
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Variable
Frame rate                               : 23.976 FPS
Minimum frame rate                       : 23.810 FPS
Maximum frame rate                       : 24.390 FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.160
Stream size                              : 1.35 GiB (100%)
Writing library                          : x264 core 142
Encoding settings                        : cabac=1 / ref=4 / deblock=1:1:1 / analyse=0x3:0x113 / me=umh / subme=8 / psy=1 / psy_rd=0.40:0.00 / mixed_ref=1 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=12 / lookahead_threads=2 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=0 / weightp=2 / keyint=96 / keyint_min=48 / scenecut=40 / intra_refresh=0 / rc_lookahead=48 / rc=2pass / mbtree=1 / bitrate=8000 / ratetol=1.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / cplxblur=20.0 / qblur=0.5 / vbv_maxrate=12000 / vbv_bufsize=18000 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=1:0.60
Codec configuration box                  : avcC
```
Audio:
```
General
Complete name                            : W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.jp.aac
Format                                   : ADTS
Format/Info                              : Audio Data Transport Stream
File size                                : 22.5 MiB
Overall bit rate mode                    : Variable

Audio
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Format version                           : Version 4
Codec ID                                 : 2
Bit rate mode                            : Variable
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 44.1 kHz
Frame rate                               : 43.066 FPS (1024 SPF)
Compression mode                         : Lossy
Stream size                              : 22.5 MiB (100%)
```
Subtitles:
```
General
Complete name                            : W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.en.ass
Format                                   : ASS
Format/Info                              : Advanced SubStation Alpha
File size                                : 45.3 KiB
FileExtension_Invalid                    : ssa

Text
Format                                   : ASS
Compression mode                         : Lossless
```

**Logs**
```
[2022-06-15 09:23:57.312 -05:00] [INF] [53] Jellyfin.Api.Helpers.TranscodingJobHelper: "C:\Program Files\Jellyfin\Server\ffmpeg.exe" "-ss 00:00:03.000 -noaccurate_seek -init_hw_device cuda=cu:0 -filter_hw_device cu -extra_hw_frames 3 -hwaccel cuda -hwaccel_output_format cuda -autorotate 0 -i file:\"W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.mp4\" -ss 00:00:03.000 -noaccurate_seek -i \"W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.jp.aac\" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 2:0 -codec:v:0 h264_nvenc -preset default -b:v 7978026 -maxrate 7978026 -bufsize 15956052 -profile:v:0 high -g:v:0 72 -keyint_min:v:0 72 -sc_threshold:v:0 0 -filter_complex \"alphasrc=s=1920x1080:r=10:start='00\:00\:03\.000',format=yuva420p,subtitles=f='W\:/webroot/jellyfin/media/anime/SPY x FAMILY/S01E01.en.ass':charenc=utf-8:alpha=1:sub2video=1,hwupload[sub];[0:0]setparams=color_primaries=bt709:color_trc=bt709:colorspace=bt709,scale_cuda=format=yuv420p[main];[main][sub]overlay_cuda=eof_action=endall:shortest=1:repeatlast=0\" -start_at_zero -codec:a:0 copy -strict -2 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 1 -hls_segment_filename \"C:\Users\owo\AppData\Local\jellyfin\transcodes\1d05ff204a2fe0ea56d95870e9c2c9c5%d.ts\" -hls_playlist_type vod -hls_list_size 0 -y \"C:\Users\owo\AppData\Local\jellyfin\transcodes\1d05ff204a2fe0ea56d95870e9c2c9c5.m3u8\""
[2022-06-15 09:23:57.588 -05:00] [ERR] [30] Jellyfin.Api.Helpers.TranscodingJobHelper: FFmpeg exited with code 1
[2022-06-15 09:23:57.689 -05:00] [ERR] [24] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request. URL "GET" "/videos/9e151d1c-4bce-134c-cc3f-455ef65d0403/hls1/main/1.ts".
MediaBrowser.Common.FfmpegException: FFmpeg exited with code 1
   at Jellyfin.Api.Helpers.TranscodingJobHelper.StartFfMpeg(StreamState state, String outputPath, String commandLineArguments, HttpRequest request, TranscodingJobType transcodingJobType, CancellationTokenSource cancellationTokenSource, String workingDirectory)
   at Jellyfin.Api.Controllers.DynamicHlsController.GetDynamicSegment(StreamingRequestDto streamingRequest, Int32 segmentId)
   at Jellyfin.Api.Controllers.DynamicHlsController.GetHlsVideoSegment(Guid itemId, String playlistId, Int32 segmentId, String container, Int64 runtimeTicks, Int64 actualSegmentLengthTicks, Nullable`1 static, String params, String tag, String deviceProfileId, String playSessionId, String segmentContainer, Nullable`1 segmentLength, Nullable`1 minSegments, String mediaSourceId, String deviceId, String audioCodec, Nullable`1 enableAutoStreamCopy, Nullable`1 allowVideoStreamCopy, Nullable`1 allowAudioStreamCopy, Nullable`1 breakOnNonKeyFrames, Nullable`1 audioSampleRate, Nullable`1 maxAudioBitDepth, Nullable`1 audioBitRate, Nullable`1 audioChannels, Nullable`1 maxAudioChannels, String profile, String level, Nullable`1 framerate, Nullable`1 maxFramerate, Nullable`1 copyTimestamps, Nullable`1 startTimeTicks, Nullable`1 width, Nullable`1 height, Nullable`1 maxWidth, Nullable`1 maxHeight, Nullable`1 videoBitRate, Nullable`1 subtitleStreamIndex, Nullable`1 subtitleMethod, Nullable`1 maxRefFrames, Nullable`1 maxVideoBitDepth, Nullable`1 requireAvc, Nullable`1 deInterlace, Nullable`1 requireNonAnamorphic, Nullable`1 transcodingMaxAudioChannels, Nullable`1 cpuCoreLimit, String liveStreamId, Nullable`1 enableMpegtsM2TsMode, String videoCodec, String subtitleCodec, String transcodeReasons, Nullable`1 audioStreamIndex, Nullable`1 videoStreamIndex, Nullable`1 context, Dictionary`2 streamOptions)
   at lambda_method656(Closure , Object )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Jellyfin.Server.Middleware.ServerStartupMessageMiddleware.Invoke(HttpContext httpContext, IServerApplicationHost serverApplicationHost, ILocalizationManager localizationManager)
   at Jellyfin.Server.Middleware.WebSocketHandlerMiddleware.Invoke(HttpContext httpContext, IWebSocketManager webSocketManager)
   at Jellyfin.Server.Middleware.IpBasedAccessValidationMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager)
   at Jellyfin.Server.Middleware.LanFilteringMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager, IServerConfigurationManager serverConfigurationManager)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.QueryStringDecodingMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.ReDoc.ReDocMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.RobotsRedirectionMiddleware.Invoke(HttpContext httpContext)
   at Jellyfin.Server.Middleware.LegacyEmbyRouteRewriteMiddleware.Invoke(HttpContext httpContext)
   at Jellyfin.Server.Middleware.ResponseTimeMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.ExceptionMiddleware.Invoke(HttpContext context)
```

**FFmpeg Logs**

```
/videos/9e151d1c-4bce-134c-cc3f-455ef65d0403/hls1/main/1.ts

{"Protocol":0,"Id":"9e151d1c4bce134ccc3f455ef65d0403","Path":"W:\\webroot\\jellyfin\\media\\anime\\SPY x FAMILY\\S01E01.mp4","EncoderPath":null,"EncoderProtocol":null,"Type":0,"Container":"mov,mp4,m4a,3gp,3g2,mj2","Size":1446252160,"Name":"S01E01","IsRemote":false,"ETag":"f3db5e0a2564a5a19b37ce723f7af260","RunTimeTicks":14499080192,"ReadAtNativeFramerate":false,"IgnoreDts":false,"IgnoreIndex":false,"GenPtsInput":false,"SupportsTranscoding":true,"SupportsDirectStream":true,"SupportsDirectPlay":true,"IsInfiniteStream":false,"RequiresOpening":false,"OpenToken":null,"RequiresClosing":false,"LiveStreamId":null,"BufferMs":null,"RequiresLooping":false,"SupportsProbing":true,"VideoType":0,"IsoType":null,"Video3DFormat":null,"MediaStreams":[{"Codec":"h264","CodecTag":"avc1","Language":"und","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/16000","CodecTimeBase":null,"Title":null,"VideoRange":"SDR","LocalizedUndefined":null,"LocalizedDefault":null,"LocalizedForced":null,"DisplayTitle":"1080p H264 SDR","NalLengthSize":"0","IsInterlaced":false,"IsAVC":false,"ChannelLayout":null,"BitRate":7978026,"BitDepth":8,"RefFrames":1,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":true,"IsForced":false,"Height":1080,"Width":1920,"AverageFrameRate":23.976004,"RealFrameRate":23.976025,"Profile":"High","Type":1,"AspectRatio":"16:9","Index":0,"Score":null,"IsExternal":false,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":false,"Path":null,"PixelFormat":"yuv420p","Level":40,"IsAnamorphic":null},{"Codec":"ass","CodecTag":null,"Language":"eng","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":null,"CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":"Undefined","LocalizedDefault":"Default","LocalizedForced":"Forced","DisplayTitle":"English - ASS","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":null,"BitRate":null,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":null,"SampleRate":null,"IsDefault":false,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":null,"Type":2,"AspectRatio":null,"Index":1,"Score":null,"IsExternal":true,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":true,"SupportsExternalStream":true,"Path":"W:\\webroot\\jellyfin\\media\\anime\\SPY x FAMILY\\S01E01.en.ass","PixelFormat":null,"Level":null,"IsAnamorphic":null},{"Codec":"aac","CodecTag":null,"Language":"jp","ColorRange":null,"ColorSpace":null,"ColorTransfer":null,"ColorPrimaries":null,"Comment":null,"TimeBase":"1/28224000","CodecTimeBase":null,"Title":null,"VideoRange":null,"LocalizedUndefined":null,"LocalizedDefault":null,"LocalizedForced":null,"DisplayTitle":"Jp - AAC - Stereo","NalLengthSize":null,"IsInterlaced":false,"IsAVC":null,"ChannelLayout":"stereo","BitRate":134769,"BitDepth":null,"RefFrames":null,"PacketLength":null,"Channels":2,"SampleRate":44100,"IsDefault":false,"IsForced":false,"Height":null,"Width":null,"AverageFrameRate":null,"RealFrameRate":null,"Profile":"LC","Type":0,"AspectRatio":null,"Index":2,"Score":null,"IsExternal":true,"DeliveryMethod":null,"DeliveryUrl":null,"IsExternalUrl":null,"IsTextSubtitleStream":false,"SupportsExternalStream":true,"Path":"W:\\webroot\\jellyfin\\media\\anime\\SPY x FAMILY\\S01E01.jp.aac","PixelFormat":null,"Level":0,"IsAnamorphic":null}],"MediaAttachments":[],"Formats":[],"Bitrate":7979828,"Timestamp":null,"RequiredHttpHeaders":{},"TranscodingUrl":null,"TranscodingSubProtocol":null,"TranscodingContainer":null,"AnalyzeDurationMs":null,"DefaultAudioStreamIndex":null,"DefaultSubtitleStreamIndex":null}

C:\Program Files\Jellyfin\Server\ffmpeg.exe -ss 00:00:03.000 -noaccurate_seek -init_hw_device cuda=cu:0 -filter_hw_device cu -extra_hw_frames 3 -hwaccel cuda -hwaccel_output_format cuda -autorotate 0 -i file:"W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.mp4" -ss 00:00:03.000 -noaccurate_seek -i "W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.jp.aac" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:0 -map 2:0 -codec:v:0 h264_nvenc -preset default -b:v 7978026 -maxrate 7978026 -bufsize 15956052 -profile:v:0 high -g:v:0 72 -keyint_min:v:0 72 -sc_threshold:v:0 0 -filter_complex "alphasrc=s=1920x1080:r=10:start='00\:00\:03\.000',format=yuva420p,subtitles=f='W\:/webroot/jellyfin/media/anime/SPY x FAMILY/S01E01.en.ass':charenc=utf-8:alpha=1:sub2video=1,hwupload[sub];[0:0]setparams=color_primaries=bt709:color_trc=bt709:colorspace=bt709,scale_cuda=format=yuv420p[main];[main][sub]overlay_cuda=eof_action=endall:shortest=1:repeatlast=0" -start_at_zero -codec:a:0 copy -strict -2 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 1 -hls_segment_filename "C:\Users\owo\AppData\Local\jellyfin\transcodes\34f27d7768181ab06ed813f9294252cc%d.ts" -hls_playlist_type vod -hls_list_size 0 -y "C:\Users\owo\AppData\Local\jellyfin\transcodes\34f27d7768181ab06ed813f9294252cc.m3u8"


ffmpeg version 5.0.1-Jellyfin Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 10-win32 (GCC) 20220113
  configuration: --prefix=/opt/ffmpeg --arch=x86_64 --target-os=mingw32 --cross-prefix=x86_64-w64-mingw32- --pkg-config=pkg-config --pkg-config-flags=--static --extra-libs='-lfftw3 -lstdc++' --extra-cflags=-DCHROMAPRINT_NODLL --extra-version=Jellyfin --disable-ffplay --disable-debug --disable-doc --disable-sdl2 --disable-ptx-compression --disable-w32threads --enable-pthreads --enable-shared --enable-lto --enable-gpl --enable-version3 --enable-schannel --enable-iconv --enable-libxml2 --enable-zlib --enable-lzma --enable-gmp --enable-chromaprint --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libass --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libwebp --enable-libvpx --enable-libzimg --enable-libx264 --enable-libx265 --enable-libdav1d --enable-libfdk-aac --enable-opencl --enable-dxva2 --enable-d3d11va --enable-amf --enable-libmfx --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
  libavutil      57. 17.100 / 57. 17.100
  libavcodec     59. 18.100 / 59. 18.100
  libavformat    59. 16.100 / 59. 16.100
  libavdevice    59.  4.100 / 59.  4.100
  libavfilter     8. 24.100 /  8. 24.100
  libswscale      6.  4.100 /  6.  4.100
  libswresample   4.  3.100 /  4.  3.100
  libpostproc    56.  3.100 / 56.  3.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'file:W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf58.78.100
  Duration: 00:24:09.91, start: 0.000000, bitrate: 7979 kb/s
  Stream #0:0[0x1](und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(progressive), 1920x1080 [SAR 1:1 DAR 16:9], 7978 kb/s, 23.98 fps, 23.98 tbr, 16k tbn (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
[aac @ 000001f47369e5c0] Estimating duration from bitrate, this may be inaccurate
Input #1, aac, from 'W:\webroot\jellyfin\media\anime\SPY x FAMILY\S01E01.jp.aac':
  Duration: 00:23:23.09, bitrate: 134 kb/s
  Stream #1:0: Audio: aac (LC), 44100 Hz, stereo, fltp, 131 kb/s
[Parsed_subtitles_2 @ 000001f47fd8ab00] libass API version: 0x1600000
[Parsed_subtitles_2 @ 000001f47fd8ab00] libass source: commit: a2b39cde4ecb74d5e6fccab4a5f7d8ad52b2b1a4
[Parsed_subtitles_2 @ 000001f47fd8ab00] Shaper: FriBidi 1.0.12 (SIMPLE) HarfBuzz-ng 4.3.0 (COMPLEX)
[Parsed_subtitles_2 @ 000001f47fd8ab00] Using font provider directwrite (with GDI)
Invalid input file index: 2.
```

**Server System (please complete the following information):**
 - OS: Windows 11
 - Jellyfin Version: 10.8.0
 - Hardware settings & device: NVENC on GTX1050
 - Reverse proxy: none
 - Other hardware notes: none
 
**Client System (please complete the following information):**
 - Device: Roku Express 4k
 - Client: Jellyfin App v1.4 build 12

**Additional**
Is almost definitley related to #7580.

I discovered that changing the `-map 2:0` to `-map 1:0` in the ffmpeg command and running it manually completes succesfully. I think that jellyfin thinks the audio stream is stream 2 because it thinks that the subtitles are stream 1 even though they are passed via a filter? (I have all subs burned in for the `roku` user because no video plays when it has soft subs, "an unexpected error occured" (or something like that), but that's a different issue I should make later.)

This issue does not affect the web client or offical Android client.

I noticed that even though my server had updated to 10.8.0, the bundled ffmpeg version was still the same (not updated to 5.0.1, I updated it manually yesterday by unzipping the 5.0.1-5 windows release from https://github.com/jellyfin/jellyfin-ffmpeg into the server directory). Could this mean that only some files/parts of jellyfin were updated, and others (like the file patched in #7580) weren't, causing the server to report version 10.8.0 even though some files were (probably) stuck in write-lock limbo (or something like that) and not updated alongside everything else?

**Comment by nyanmisaka** [comment]
Should be fixed by the PR linked above.

**Comment by helpimnotdrowning** [comment]
I fixed it by running the 10.8.0 stable installer. Even thought the webui said it was running version 10.8.0, it seems like it wasn't *actually* 10.8.0 (maybe it was one of the unstable builds?). 

(also, I was under the assumption there was an auto-updater (I probably misread something), but after looking into it more it seems there never was one in the first place.)

**Comment by nyanmisaka** [comment]
Thanks anyway. The #7580 is only a partial fix but should cover most cases.
